packages/next-polyfill-nomodule/src/index.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable import/no-extraneous-dependencies */
-// Keep in sync with eslint no-unwanted-polyfillio rule: https://github.com/vercel/next.js/blob/master/packages/eslint-plugin-next/lib/rules/no-unwanted-polyfillio.js
+// Keep in sync with eslint no-unwanted-polyfillio rule: https://github.com/vercel/next.js/blob/master/packages/eslint-plugin-next/lib/rules/no-unwanted-polyfillio.ts
```

Comment:
```suggestion
// Keep in sync with eslint no-unwanted-polyfillio rule: https://github.com/vercel/next.js/blob/master/packages/eslint-plugin-next/lib/rules/no-unwanted-polyfillio.js
```

Reply:
also change `master` to `main` (maybe out of scope for this pr)